content/test/integration/E2ETestUtils.ts

```diff
@@ -122,8 +122,8 @@ export function awaitUntil(
 export async function deployEntitiesCombo(
   service: MetaverseContentService,
   ...entitiesCombo: EntityCombo[]
-): Promise<Timestamp> {
-  let timestamp: Timestamp = 0
+): Promise<Timestamp | ErrorList> {
+  let timestamp: Timestamp | ErrorList = []
```

Comment:
I'm curious, why the change to an empty list by default?

Reply:
I wanted to reflect that the default value is error instead of a successful state to avoid false positives 🤔

Reply:
Oh, ok. Makes sense, good point!